### ISSUE TITLE
html logger not replacing template variables

### ISSUE THREAD

**Issue by chriskacerguis** [jamesoff/simplemonitor#550]
Hello!  Thanks for all your efforts with this great library.

I'm running into an issue with the html logger, I did a fresh install `pip install simplemonitor` and I am using nginx:alpine to show the HTML.  The page is loading, but I'm seeing the following errors (also see screenshot).

```
[Error] ReferenceError: Can't find variable: _INTERVAL_
	check_age (status.html:22)
	onload (status.html:30)

[Warning] jQuery.Deferred exception: Can't find variable: _INTERVAL_ – "check_age@http://REDACTED:8081/s/status.html:22:31↵http://REDACTED:8081/s/status.html:65:1…" – undefined (jquery-3.4.1.slim.min.js, line 2)

[Error] ReferenceError: Can't find variable: _INTERVAL_
	(anonymous function) (jquery-3.4.1.slim.min.js:2:31330)
```

<img width="1613" alt="Screen Shot 2020-06-06 at 12 02 44 PM" src="https://user-images.githubusercontent.com/2414647/83950086-aa089b80-a7ed-11ea-9142-54a56299acb8.png">

The page is refreshing at the interval I set (15), so I know it is working...I'm just not sure what the issue is with the template vars.  I don't support you have an idea of what I am doing wrong?

A few more details, comparing the code vs what is generated SOME template variables are working.  Here is the code vs. what is rendered in my browser (latest Safari)

https://github.com/jamesoff/simplemonitor/blob/develop/simplemonitor/html/header.html#L7 

```html
<title>OK@5d77549f5fa3 Monitor</title>
```

https://github.com/jamesoff/simplemonitor/blob/develop/simplemonitor/html/header.html#L11

```html
<meta http-equiv="refresh" content="_INTERVAL_" />
```

https://github.com/jamesoff/simplemonitor/blob/develop/simplemonitor/html/header.html#L21

```html
var generated = 1591463515;
```

https://github.com/jamesoff/simplemonitor/blob/develop/simplemonitor/html/header.html#L24

`if (diff > (_INTERVAL_ * 2)) {`

https://github.com/jamesoff/simplemonitor/blob/develop/simplemonitor/html/header.html#L33

```html
<div class="navbar navbar-light bg-light border-bottom OKBORDER_" id="summary">
```

https://github.com/jamesoff/simplemonitor/blob/develop/simplemonitor/html/header.html#L35

```html
<div class="navbar-text d-none d-lg-block" id="updated">
Updated at 2020-06-06 17:11:55 by 5d77549f5fa3 (_VERSION_)
</div>
```

https://github.com/jamesoff/simplemonitor/blob/develop/simplemonitor/html/header.html#L37

```html
<div class="navbar-text">
  <div id="summary" class="ok">
    OK
  <div id="details">
    <span class="ok">1 OK</span>
    <span class="fail">0 FAIL</span>
    <span class="old">0 OLD</span>
    <span class="remote">0 remote</span>
  </div>
</div>
</div>
```

Here is the generated HTML table contents

```html
<tr class="okrow">
  <td class="monitor_name">localhost</td>
  <td class="status ok">OK</td>
  <td>5d77549f5fa3</td>
  <td></td>
  <td class="vfc">&nbsp;</td>
  <td>0+00:00:00</td>
  <td> &nbsp;</td>
  <td></td>
  <td></td>
  <td></td>
</tr>
```

https://github.com/jamesoff/simplemonitor/blob/develop/simplemonitor/html/footer.html#L2

```html
<div class="navbar-text d-block d-lg-none" id="updated">
Updated at 2020-06-06 17:20:03<br />by 5d77549f5fa3 (_VERSION_)
</div>
```

Thanks

**[Edit]:** cleanup and details

**Comment by jamesoff** [comment]
Hi! You're welcome :)

Can you confirm what version got installed by pip?

That does look like the replacement isn't working right for the interval somewhere in the page. Could you have a look at the source for the page? The interval should show up in a `<meta>` refresh at the top, and inside the `check_age()` not far below it, which from the error looks like the one that's getting missed.

**Comment by chriskacerguis** [comment]
updated with more info...checking version

**Comment by chriskacerguis** [comment]
```sh
bash-5.0# pip show simplemonitor
Name: simplemonitor
Version: 1.9.0
Summary: A simple network and host monitor
Home-page: https://github.com/jamesoff/simplemonitor
Author: James Seward
Author-email: james@jamesoff.net
License: UNKNOWN
Location: /usr/local/lib/python3.7/site-packages
Requires: requests, pyOpenSSL, colorlog, psutil, paho-mqtt, boto3
Required-by: 
bash-5.0# 
```

**Comment by jamesoff** [comment]
Cheers for the info. Something definitely not right there, I recall fixing something similar but thought it was after 1.9.0. Let me test it out on a fresh install...

**Comment by jamesoff** [comment]
If you want the newer HTML style, you can upgrade your install to the `develop` branch like this btw:

```
pip install --upgrade git+https://github.com/jamesoff/simplemonitor@develop
```

That branch is stable, my environment is generally running it (or a feature branch to validate it before I merge), and there's not much I want to do now before 1.10.

**Comment by chriskacerguis** [comment]
awesome.  thanks so much for your help with this and taking some of your free time to take a look.

**Comment by Barna-154** [comment]
hie how do i setup the html dashboard view i am failing to get it

**Comment by jamesoff** [comment]
> hie how do i setup the html dashboard view i am failing to get it

@Barna-154 could you please open a new issue for this? Please include your `monitor.ini` config and any errors you're getting. Thanks!

**Comment by jamesoff** [comment]
@chriskacerguis going to close this. Please re-open if you're still having this issue.
